### PR TITLE
Allow to have nothing selected on Select

### LINKED ISSUE #925: Select does not let you have nothing selected
_via @astanciu_

**Describe the bug**
When using the Select component, you always need to have an item as the current selected one.

**Expected behavior**
By default nothing is selected.

### PR DESCRIPTION
**Pull request by francocorreasosa**
## Default
image

## With custom empty option text
image

Fixes #925

### CODE REVIEW

File: core/components/atoms/select/select.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ const Select = ({ options, ...props }) => {
 
   return (
     <StyledSelect {...props} {...Automation('select')}>
+      <option disabled selected={!props.value} value="">
```
Comment (siddharthkp) [inline core/components/atoms/select/select.js]: what does `!props.value` do 🤔

Comment (francocorreasosa) [reply core/components/atoms/select/select.js]: If we have a `props.value` set, this won't be the selected option, the `value` value will be

Comment (siddharthkp) [reply core/components/atoms/select/select.js]: 😅 got it

can you leave a comment there, looks like it has something to do with boolean values !false = true

Comment (francocorreasosa) [reply core/components/atoms/select/select.js]: Sure thing! Done.

**Pull request merged**